src/workerd/api/queue.h

```diff
@@ -64,7 +82,9 @@ class WorkerQueue: public jsg::Object {
     JSG_TS_ROOT();
     JSG_TS_OVERRIDE(Queue<Body = unknown> {
       send(message: Body, options?: QueueSendOptions): Promise<void>;
-      sendBatch(messages: Iterable<MessageSendRequest<Body>>): Promise<void>;
+      sendBatch(messages
+                : Iterable<MessageSendRequest<Body>>, options ?: QueueSendBatchOptions)
```

Comment:
What's going on with the whitespace in here? Have you managed to get `make format` working?

Reply:
There is no `make format` in this project. In the `edgeworker` repo, `make format` uses `clang-format` with a custom code style; when I apply it to this file, I get the same result (same line breaking and space before `?`.